### DISCUSSION TITLE
Cannot download my roommate's package

### DISCUSSION THREAD

**Discussion by mikoV97** [IBAHCM/RPiR#228] [Archive] [answered]
Hi @richardreeve @soniamitchell 
I try to check my roommates' code, I can download Jennifer's but I cannot download ben's.
I got a warning message like this when I run the code devtools::install_github("IBAHCM/benmcgeachieSeries03"):

Error : Cannot detect that project is already a Git repository.
Do you need to run `use_git()`?
: unable to load R code in package ‘benmcgeachieSeries03’
ERROR: lazy loading failed for package ‘benmcgeachieSeries03’
* removing ‘/Library/Frameworks/R.framework/Versions/4.0/Resources/library/benmcgeachieSeries03’
: Failed to install 'benmcgeachieSeries03' from GitHub:
 installation of package ‘/var/folders/zs/r2gl8n_s32qfmrcd1xmq3sm00000gn/T//Rtmpopw0K8/file150fd3c4fb798/benmcgeachieSeries03_0.1.0.tar.gz’ had non-zero exit status
: There were 21 warnings (use warnings() to see them)

I am using a Mac, and I remembered yesterday someone had the same issue, but I can not find out where to remove the file as the warnings says, and I also try to search the file, but failed.

**Comment by soniamitchell** [comment, marked as answer]
Hey @mikoV97, try asking him to delete R/0301_setup.R (it's not a function!) and `devtools::document()` again, then push. Then try again.

**Reply by mikoV97** [reply]
Hi Sonia! Thank you so much! It works now!!!!
